nix/pkgs/mayastor/cargo-package.nix

```diff
@@ -83,15 +87,25 @@ let
   };
 in
 {
-  release = rustPlatform.buildRustPackage (buildProps // {
-    cargoBuildFlags = "--bin mayastor --bin mayastor-client --bin mayastor-csi";
-    buildType = "release";
-    buildInputs = buildProps.buildInputs ++ [ libspdk ];
-    SPDK_PATH = "${libspdk}";
-  });
-  debug = rustPlatform.buildRustPackage (buildProps // {
-    buildType = "debug";
-    buildInputs = buildProps.buildInputs ++ [ libspdk-dev ];
-    SPDK_PATH = "${libspdk-dev}";
-  });
+  release = rustPlatform.buildRustPackage
```

Comment:
Why is the indentation changed here?

Reply:
fixed https://github.com/openebs/mayastor/pull/1041/commits/778fc7c2c74269b75e51c5e6f809f8a4466576e6